.github/workflows/create_release.yml

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Caller Workflow
+on: 
+  schedule:
+    # Run weekly on Monday 00:00
+    - cron:  '00 00 * * MON'
+  push:
+    branches: [main, rel-*]
+  pull_request:
+    branches: [main, rel-*]
+  workflow_dispatch:    
+
+jobs: 
+ 
+  call-workflow-sourcedist:    
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']    
+    uses: ./.github/workflows/release_sourcedist.yml    
+    with:
+      os: "mac" 
+
+  call-workflow-ubuntu_x86:    
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']    
+    uses: ./.github/workflows/release_linux_x86_64.yml   
+    with:            
+      os: "linux_x86_64" 
+
+  call-workflow-ubuntu_aarch64:    
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']    
+    uses: ./.github/workflows/release_linux_aarch64.yml
+    with:            
+      os: "linux_aarch64"      
+
+  call-workflow-win:    
+    strategy:
+      matrix:
+        os: ['windows-latest']    
+    uses: ./.github/workflows/release_win_x86_64.yml
+    with:      
+      os: "win"      
+
+  call-workflow-mac:    
+    strategy:
+      matrix:
+        os: ['mac-latest']    
+    uses: ./.github/workflows/release_mac.yml
+    with:            
+      os: "mac"            
+
+  publish-weekly:
+    # Publish weekly pypi, if not all OS variants are successful, it's ok
+    name: Publish Weekly to PyPI 
+    runs-on: ubuntu-latest
+    needs: [call-workflow-mac, call-workflow-ubuntu_x86, call-workflow-ubuntu_aarch64, call-workflow-win, call-workflow-sourcedist]
+    # does not run on forks or pull requests
+    if: (github.repository_owner == 'onnx') && (( github.event_name == 'schedule' )) && ((needs.call-workflow-mac.result == 'success') || (needs.call-workflow-ubuntu_x86.result == 'success') || (needs.call-workflow-ubuntu_aarch64.result == 'success') || (needs.call-workflow-sourcedist.result == 'success'))
+
+    environment:
+      name: pypi
+      url: https://pypi.org/p/onnx-weekly/
```

Comment:
this does not create anything. It's a environment configuration for trusted publishing: At pypi you can specify the environviroment which is allowed to publish.
<img width="476" alt="image" src="https://github.com/user-attachments/assets/07a8f86c-926f-46f3-8fab-9d6d668787ac">

yes, handling "onnx-weekly" is already implemented / shifted

I don't intend to automate the deletion, I think it can be done manually.